### ISSUE TITLE
Configuring Alfen Wallbox not working, weird login info

### ISSUE THREAD

**Issue by jaques-sam** [leeyuentuen/alfen_wallbox#133]
It's not really clear from the installation description what the login information should be.
Typically, one should have a "User level" which is either one of these 3:
- Owner
- Temporary
- Secure service access
and a password. Just like the ACE Service Installer requires to access an Alfen device.

I have tried the "Owner" + password, which returns this error:
Error: "Unexpected error creating device."

So what the h*** is the username here? 

![image](https://github.com/leeyuentuen/alfen_wallbox/assets/42445148/7b5f2b46-48a2-40f0-be28-4d2881db3bab)

Sorry if this sounds nooby, but please update the documentation! I'm probably not the only one with this issue ;-)

**Comment by leeyuentuen** [comment]
username should admin

**Comment by jaques-sam** [comment]
Yeah I know :-) That's also very unclear from the doc, but I could find it in other issues reported here.

**Comment by phulstaert** [comment]
I think you found your answer [here](https://github.com/leeyuentuen/alfen_wallbox/issues/83#issuecomment-2221287449)? If so, you can mark this as 'Done'.

**Comment by jaques-sam** [comment]
Sorry Peter, the reason I opened that ticket was about the poor documentation…______________On 11 Jul 2024, at 00:33, Peter Hulstaert ***@***.***> wrote:﻿
I think you found your answer here? If so, you can mark this as 'Done'.

**Comment by leeyuentuen** [comment]
> Sorry Peter, the reason I opened that ticket was about the poor documentation…______________On 11 Jul 2024, at 00:33, Peter Hulstaert ***@***.***> wrote:﻿ I think you found your answer here? If so, you can mark this as 'Done'. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

be my guest to write some documentation with your experience. that can save each other times

**Comment by fwestenberg** [comment]
Documentation is updated already.
